generators/import-jdl/index.js

```diff
@@ -33,7 +33,8 @@ function importJDL() {
   const jdlImporter = jhiCore.JDLImporter.createImporterFromFiles(this.jdlFiles, {
     databaseType: this.prodDatabaseType,
     applicationType: this.applicationType,
-    applicationName: this.baseName
+    applicationName: this.baseName,
+    forceNoFiltering: true
```

Comment:
Why would updating JSON content? why not before?

Reply:
forceNoFiltering just ignore files that didn't changed. Or I am wrong?

See https://github.com/oktadeveloper/generator-jhipster-ionic/issues/279#issuecomment-604111842

Look at the logs:
```
INFO! Found entities: Blog, Entry, Tag.
INFO! The JDL has been successfully parsed.
INFO! Generating 3 entities.

Found the .jhipster/Blog.json configuration file, entity can be automatically generated!

The entity Blog is being updated.

Found the .jhipster/Entry.json configuration file, entity can be automatically generated!

The entity Entry is being updated.

Found the .jhipster/Tag.json configuration file, entity can be automatically generated!

The entity Tag is being updated.
```

The entities files are there, the problem is that jhipster-core isn't returning.
So my guess is that jhipster-core isn't returning because the file didn't change.

I suppose the use case is removing the src folder but not the .jhipster folder, causing this bug.

I really think ionic should use the original .jhipster folder, but this is another discussion.

Reply:
> The entities files are there, the problem is that jhipster-core isn't returning.

The returned state returns what is to be written, always. If it doesn't, that's a huge issue...

> I suppose the use case is removing the src folder but not the .jhipster folder, causing this bug.
> I really think ionic should use the original .jhipster folder, but this is another discussion.

Agreed

Reply:
> The entities files are there, the problem is that jhipster-core isn't returning.

If there's a way I can write a test in JCore for this, that'd confirm and fix it

Reply:
> > The entities files are there, the problem is that jhipster-core isn't returning.
> 
> The returned state returns what is to be written, always. If it doesn't, that's a huge issue...

I think the file is not written because the file was already there.
So the entity was not returned.

I don't think this is a jhipster-core bug.
It's the assumption that the jhipster-core will always return the entities.
With forceNoFiltering it will always return.

Reply:
The log (with no generation) happens when there is no `.jhipster` folder in the Ionic project. If there is a folder, it spits out the proper message that nothing has changed and therefore, no generation should happen.

The strange thing was everything worked fine in CI yesterday before releasing and it does with this PR too. However, when I tried `npm link` locally, I was able to reproduce the issue. I'll try this change - thanks!